Week2/todo_app/program.js

```diff
@@ -21,52 +18,159 @@ class TodoModel {
         });
     }
 
-    create(description, callback) {
-        // Write code and query to create a new TODO item
+    create(description, user_id, callback) {
+        const createTodoItem = `INSERT INTO todo_items(text, is_completed, user_id) VALUES(${dbConnection.escape(description)}, 0, ${dbConnection.escape(user_id)})`;
```

Comment:
This is a very long line - it doesn't even fit in the window in github. Since you're using backticks for string templating, it's easy to break this up into a multi-line string - and SQL ignores newlines.

Nice job with the query, and great job using escaping!